### PR TITLE
Add browser support

### LINKED ISSUE #60: Browser support?
We can use `document.execCommand` or the Async Clipboard API to create a 1:1 implementation for browsers.

See: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

cc @sindresorhus

### PR DESCRIPTION
**Pull request by Richienb**
Fixes #60

cc @sindresorhus

### CODE REVIEW

**Comment by sindresorhus** [comment]
> See https://nodejs.org/api/esm.html#esm_conditional_exports on how to add the browser version.

**Comment by Richienb** [comment]
@sindresorhus Should I be concerned that browserify doesn't detect the bundle this way?

**Comment by sindresorhus** [comment]
We can have both the `browser` field and the conditional exports. I would recommend opening an issue on Browserify about supporting conditional exports though. That's the future.

**Comment by Richienb** [comment]
@sindresorhus Done!

File: package.json [hunk 1]
```diff
@@ -43,5 +44,23 @@
 		"ava": "^2.1.0",
 		"tsd": "^0.10.0",
 		"xo": "^0.25.3"
+	},
+	"browser": "browser.js",
+	"exports": {
+		".": {
```
Comment (sindresorhus) [inline package.json]: > If the "." export is the only export, the "exports" field provides sugar for this case being the direct "exports" field value.

File: browser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
```
Comment (sindresorhus) [inline browser.js]: Easier to just add `/* eslint-globals navigator */` than the XO overrides in package.json.

File: browser.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+exports.write = async text => {
+	await navigator.clipboard.writeText(text);
+};
+
+exports.read = async () => {
+	return navigator.clipboard.readText();
+};
+
+exports.readSync = () => {
+	throw new Error('readSync not supported in browsers!');
```
Comment (sindresorhus) [inline browser.js]:
```suggestion
	throw new Error('`.readSync()` is not supported in browsers!');
```

File: package.json (hunk 1 above)
Comment (Richienb) [reply package.json]: Fixed!

File: browser.js (hunk 2 above)
Comment (Richienb) [reply browser.js]: Fixed!

File: browser.js (hunk 3 above)
Comment (Richienb) [reply browser.js]: Applied!

**Comment by sindresorhus** [comment]
Since the browser version is just for modern browsers, we can just use ES2015 export syntax there.

**Comment by Richienb** [comment]
Fixed!

**Comment by Richienb** [comment]
@sindresorhus Please re-review.

**Pull request merged**